pkgs/by-name/wc/wcurl/package.nix

```
@@ -0,0 +1,42 @@
+{ stdenv
+, lib
+, curl
+, fetchFromGitHub
+, makeWrapper
+}:
+
+stdenv.mkDerivation rec {
+  pname = "wcurl";
+  version = "2024-07-07";
+
+  src = fetchFromGitHub {
+    owner = "Debian";
+    repo = pname;
+    rev = version;
```

Use `refs/tags` and replace `rec` with `finalAttrs`
```suggestion
    rev = "refs/tags/${finalAttrs.version}";
```